<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Confirmation Procedure</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TITLE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1021</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Unispace</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>New injury again 
 or 
 finished?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="new_injury_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>331</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New Injury</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finish_procedure">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>350</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Finished</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
